docs/0.3-DRAFT/index.md

```diff
@@ -588,7 +717,7 @@ If there is no web-accessible description for a file format it SHOULD be describ
     "@type": "File",
     "contentSize": "120",
     "description": "A file in a non-standard format",
-    "encodingFormat": ["text/plain", {"@id": "https://www.commonwl.org/v1.0/Workflow.html"}]
+    "encodingFormat": ["text/plain", {"@id": "some_extension.md"}]
```

Comment:
Good correction - as indeed CWL has version of its own language as REQUIRED in every `*.cwl` file ;) 

```yaml
#!/usr/bin/env cwl-runner
cwlVersion: v1.0
class: Workflow
```

which is documented at permalink <https://w3id.org/cwl/v1.0/> which then would be a natural `@id` to use for `encodingFormat` (shown in Workflow section I think)